lib/contracts/formatters.rb

```diff
@@ -85,8 +85,12 @@ def plain?
 
       def useful_to_s?
         # Useless to_s value or no custom to_s behavious defined
-        # Ruby < 2.0 makes inspect call to_s so this won't work
-        @value.to_s != '' && @value.to_s != @value.inspect
+        @value.to_s != '' &&
+          if @value.class == Class # It's a class contract
```

Comment:
multiline `if..else..end` as a second condition to complex condition after `&&` looks confusing to me. Any chance extracting `if..else..end` part to method?